setup/backup.class.inc.php

```diff
@@ -265,35 +265,44 @@ protected function PrepareFilesToBackup($sSourceConfigFile, $sTmpFolder, $bSkipS
 			SetupUtils::copydir($sExtraDir, $sFile);
 			$aRet[] = $sFile;
 		}
+
+		$aExtraFiles = [];
 		if (MetaModel::GetConfig() !== null) // During unattended install config file may be absent
 		{
 			$aExtraFiles = MetaModel::GetModuleSetting('itop-backup', 'extra_files', []);
-			foreach($aExtraFiles as $sExtraFileOrDir)
+		}
+
+		/** @var iBackupExtraFilesExtension $oExtensionInstance */
+		foreach (MetaModel::EnumPlugins('iBackupExtraFilesExtension') as $oExtensionInstance)
+		{
+			$aExtraFiles = array_merge($aExtraFiles, $oExtensionInstance->GetExtraFiles());
+		}
```

Comment:
```suggestion
		foreach (utils::GetClassesForInterface('iBackupExtraFilesExtension', '', ['[\\\\/]lib[\\\\/]', '[\\\\/]node_modules[\\\\/]', '[\\\\/]tests[\\\\/]']) as $sExtensionClass)
		{
			/** @var iBackupExtraFilesExtension $oExtensionInstance */
			$oExtensionInstance = new $sExtensionClass();
			$aExtraFiles = array_merge($aExtraFiles, $oExtensionInstance->GetExtraFiles());
		}
```

Rather use this method to retrieve classes implementing the API, it has 2 main advantages (and 1 drawback):
- No need to declare the API interface in `\MetaModel::InitExtensions()`
- No need to explicit require class in the `module.xxx.php` file, it can be autoloaded

_The drawback is that you get FQCN of classes implementing the interface instead of an instance of these classes_